<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/imagenes/recursos/icono.png</normaloff>:/imagenes/recursos/icono.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Centigrados</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLCDNumber" name="outCent">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 105, 97);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="cmdCent">
         <property name="minimum">
          <number>-18</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Fahrenheit</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLCDNumber" name="outFahr">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(102, 192, 11);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="cmdFahr">
         <property name="maximum">
          <number>210</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Kelvin</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLCDNumber" name="outKel">
         <property name="styleSheet">
          <string notr="true">
background-color: rgb(119, 158, 203);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="cmdKel">
         <property name="minimum">
          <number>255</number>
         </property>
         <property name="maximum">
          <number>373</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cmdCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outFahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdKel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outKel</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
